packages/account-v2/src/components/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useCopyToClipboard } from 'usehooks-ts';
+import { LabelPairedCircleCheckCaptionFillIcon, StandaloneCopyRegularIcon } from '@deriv/quill-icons';
+import { Tooltip, useDevice } from '@deriv-com/ui';
+
+type TProps = {
+    infoMessage?: string;
+    popoverAlignment?: 'bottom' | 'left' | 'right' | 'top';
+    successMessage?: string;
+    textCopy: string;
+};
+
+/**
+ * @depricated TODO: remove this when it's available in @deriv-com/ui. This is temporary.
+ */
+const Clipboard: React.FC<TProps> = ({
+    infoMessage = 'Copy',
+    popoverAlignment,
+    successMessage = 'Copied',
+    textCopy,
+}) => {
+    const [, copy] = useCopyToClipboard();
+    const [isCopied, setIsCopied] = useState(false);
+    const { isMobile } = useDevice();
+    const timeoutClipboard = useRef<ReturnType<typeof setTimeout>>();
+
+    const onClick = () => {
+        setIsCopied(true);
+        copy(textCopy);
+        timeoutClipboard.current = setTimeout(() => {
+            setIsCopied(false);
+        }, 2000);
+    };
+
+    useEffect(() => {
+        return () => clearTimeout(timeoutClipboard.current);
+    }, []);
+
+    return (
+        <Tooltip
+            message={isCopied ? successMessage : infoMessage}
+            position={popoverAlignment}
+            triggerAction={isMobile ? 'click' : 'hover'}
+        >
+            <button className='flex items-center cursor-pointer px-0 mx-0 border-0 w-fit' onClick={onClick}>
+                {isCopied ? (
+                    <LabelPairedCircleCheckCaptionFillIcon fill='#4bb4b3' height={24} width={24} />
```

Comment:
There is a tailwind color configured for this Hex code. Instead of fill property u can use `className='fill-status-light-success'`